utils/todo.go

```diff
@@ -0,0 +1,78 @@
+package utils
+
+import (
+	"cronicle/ui/constants"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func GetTodoFromFile(f fs.FileInfo) []string {
+	path := GetPath([]string{"todo", f.Name()})
+	dat, _ := os.ReadFile(path)
+	fileArr := strings.Split(string(dat), "\n")
+	return fileArr
+}
+
+func GetAllTodos() []fs.FileInfo {
+	dirPath := GetPath([]string{"todo"})
+	files, _ := ioutil.ReadDir(dirPath)
+	return files
+}
+
+func ComposeTodo(w WriteParams) []string {
+	output := make([]string, 0)
```

Comment:
Instead of building a `[]string`, let's use `strings.Builder`. Something like this:

```go
s := strings.Builder{}

s.WriteString("---")
...
...

s.WriteString("- [ ] Le todoo

return s.String()
```